src/steamship/app/invocable_request.py

```diff
@@ -36,7 +27,7 @@ class InvocationContext(CamelModel):
     invocable_type: str = None
 
 
-class Request(CamelModel):  # TODO (enias): Rename to ExecutableRequest?
```

Comment:
Renamed so to resolve conflict with base.request.Request.

Reply:
Really glad to see this -- if we can I think probably good to avoid any naming duplicates in our code base